src/routes/pdf-fetcher/containers/PdfFetcherPage.js

```diff
@@ -154,7 +151,7 @@ class PdfFetcherPage extends React.Component {
 const mapStateToProps = (state) => ({
   location: state.router.params.location,
   language: state.router.params.language,
-  path: state.router.params['_'] // _ contains all the values from *
+  fetchUrl: state.router.query.url
```

Comment:
Why did you change this to query.url ? 🤔

Reply:
because we have a whole url now to identify a category (see comments above why url to category map is useful) and so imo it looks better with a query parameter, but maybe this is just a question of personal preference. also for me the url is just /location/language/fetch-pdf, and the query url is just a parameter

Reply:
It's alright to me, but we shouldn't change urls too often. Since it's just the pdf-fetch-page it's not a big deal though.

Reply:
jop, so this is okay?